cli/remotebazel/remotebazel.go

```diff
@@ -1134,3 +1128,15 @@ func contains(m map[string]string, elem string) bool {
 	_, ok := m[elem]
 	return ok
 }
+
+// Filters out the flags, for a simplified representation of the command.
+func getSimplifiedCommand(bazelArgs []string) string {
+	filteredArgs := make([]string, 0)
+	for _, arg := range bazelArgs {
+		if !strings.HasPrefix(arg, "--") {
+			filteredArgs = append(filteredArgs, arg)
+		}
+	}
+
+	return strings.Join(filteredArgs, " ")
+}
```

Comment:
I think this doesn't work for things like `bazel run -c opt //foo`? Can you use `parser.GetBazelCommandAndIndex` here to get the bazel command?

For the target pattern(s), it's probably not super straightforward to parse those, but you could also add a func like `parser.GetPatterns()` that does something basic for now (like any flag that doesn't start with `-` and  has `//` or `:` can be treated as a target) and add a TODO there to parse it properly.

(More generally, I think ideally we'd consolidate any parsing code into `parser.go` so that all of the hacks/known issues with flag parsing are consolidated in a central place rather than scattered in various places and hard to track down.)

Reply:
Done